.github/workflows/test-linux.yml

```
@@ -77,31 +77,36 @@ jobs:
   azure-tests:
     runs-on: ubuntu-latest
     needs: [ 'format', 'tests' ]
-    if: ${{ false }}
+    if: ${{ true }}
```

```suggestion
    if: ${{ false }}
```

revert to previous state